book/website/communication/dif-articles/reg.md

```diff
@@ -0,0 +1,74 @@
+(cm-dif-articles-registered-reports)=
+## What are Registered Reports?
+Registered Reports are an article type divided into two published parts- a study protocol and a research article. 
+The study protocol is reviewed **prior** to the data being collected and the research being carried out. 
+
+(image to go here)
```

Comment:
``````{figure} ../../figures/cos-reg-report.png
---
height: 500px
name: Registered Reports 
alt: image showing the different stages of a registered report, including stage 1 peer review after design of the study and then stage 2 peer review after the study has been conducted. 
---
This image is by the Centre for Open Science (https://www.cos.io/initiatives/registered-reports) and published under a CC BY ND 4.0 license.
```
```